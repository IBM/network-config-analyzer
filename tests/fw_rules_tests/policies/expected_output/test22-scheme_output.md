|query|src_ns|src_pods|dst_ns|dst_pods|connection|
|---|---|---|---|---|---|
|connectivity_map, config: np22|
||[default]|[test in (A,B)]|[default]|[app in (skydive)]|All connections|


