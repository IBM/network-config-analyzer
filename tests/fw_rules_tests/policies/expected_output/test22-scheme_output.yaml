- query: connectivity_map
  configs:
  - np22
  numerical_result: 0
  explanation:
  - rules:
    - src_ns:
      - default
      src_pods:
      - test!=C
      dst_ns:
      - default
      dst_pods:
      - app=skydive
      connection:
      - All connections